src/Core/Domain/Address/Command/AbstractEditAddressCommand.php

```diff
@@ -238,7 +238,7 @@ public function getCountryId(): ?CountryId
     }
 
     /**
-     * @param int|null $countryId
+     * @param int $countryId
```

Comment:
But it can be `null` sometimes 😮 
Maybe PHPStan has another way to define nullable properties, but writing `@param int $countryId` would be lying. This field can be null in some expected usecases.

Reply:
No it can't be null

`public function setCountryId(int $countryId)` And the file is in strict type mode.

Reply:
Euh yes it can be null 😛 look
<img width="343" alt="Capture d’écran 2020-08-03 à 16 32 31" src="https://user-images.githubusercontent.com/3830050/89193949-fe8a8600-d5a6-11ea-8ab1-858a6340fd5e.png">

Nobody obliges you to **call** `setCountryId(int $countryId)`. It's used when there is data 😄 . No data, no setter, it's null. If `EditCustomerAddressCommand` is built without calling the setter, `$countryId` is null. Please trust me 😄 

```
        $editAddressCommand = new EditCustomerAddressCommand($addressId);
        dump($editAddressCommand);
```